src/player.ts

```diff
@@ -28,6 +28,81 @@ export interface PlayerConfig extends NativeInstanceConfig {
    * ```
    */
   licenseKey?: string;
+  /**
+   * Configuration options for player's default playback behavior.
+   */
+  playbackConfig?: PlaybackConfig;
+}
+
+/**
+ * Configuration options for player's default playback behavior.
```

Comment:
Please use either the iOS or the Android SDKs documentation for public API documentation for this library

```suggestion
 * Configures the playback behaviour of the player.
```